engine/gamesys/src/gamesys/test/test_gamesys.h

```diff
@@ -166,6 +167,20 @@ class DrawCountTest : public GamesysTest<DrawCountParams>
     virtual ~DrawCountTest() {}
 };
 
+struct BoxRenderParams
+{
+    const char* m_GOPath;
+    dmGameSystem::BoxVertex m_ExpectedVertices[16];
+    uint8_t m_ExpectedVerticesCount;
+    int m_ExpectedIndices[54];
```

Comment:
"54" needs clarification.
Is it a product of other, more readable, constants?

Reply:
Fixed in: https://github.com/defold/defold/pull/4921/commits/ee8b54705e82bd3f14a3c75619f23bdeaf0f2fa0